### PR TITLE
Add step to verify kubeapi-server pods after updating API server certificate

### PR DESCRIPTION
**Pull request by bparry02**
This additional step helps show the reader which pods to monitor for changes, and verify that the API server pods are ready to be tested for the new cert.

I believe this should apply to all OCP 4.x branches.

### CODE REVIEW

**Comment by bparry02** [comment]
@bergerhoffer @kalexand-rh any updates on this? Thanks.

**Comment by bergerhoffer** [comment]
@sanchezl Can you weigh in on adding this step to check that the Kubernetes API server pods are ready before the cert will be ready to use? 

From this doc here: https://docs.openshift.com/container-platform/4.5/security/certificates/api-server.html#customize-certificates-api-add-named_api-server-certificates

Is this the best method for verification here? We have a few other methods in other procedures after an api server update, not sure if either would be applicable here or not:

* https://docs.openshift.com/container-platform/4.6/nodes/nodes/nodes-nodes-audit-config.html#configuring-audit-policy_nodes-nodes-audit-config (username: redhat; password: redhat)
* https://docs.openshift.com/container-platform/4.6/authentication/configuring-internal-oauth.html#oauth-token-inactivity-timeout_configuring-internal-oauth (username: redhat; password: redhat)

**Comment by sanchezl** [comment]
Checking the cluster-operator's `Progressing` state as [shown here](https://docs.openshift.com/container-platform/4.6/authentication/configuring-internal-oauth.html#oauth-token-inactivity-timeout_configuring-internal-oauth) is the best "official" answer.

**Comment by bparry02** [comment]
@bergerhoffer I have modified this PR to use the suggested verification of the kube-apiserver clusteroperator and rebased.

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

Note to self: when cherrypicking, adjust the version in the example output as appropriate.

**Comment by bergerhoffer** [comment]
@wangke19 or @barleyer Could one of you provide a QE review please? Thanks!

Preview: https://deploy-preview-21205--osdocs.netlify.app/openshift-enterprise/latest/security/certificates/api-server.html

File: modules/customize-certificates-api-add-named.adoc [hunk 1]
```diff
@@ -78,3 +78,21 @@ spec:
         name: <secret>
 ...
 ----
+
+. Check that a new revision of the Kubernetes API server pods has rolled out. This will take several minutes.
++
+[source,terminal]
+----
```
Comment (wangke19) [inline modules/customize-certificates-api-add-named.adoc:85]: After Updated the API server, the kube-apiserver cluster operator `PROGRESSING` condition will change to `True` about one minute later, before this, `PROGRESSING` condition is still 'False`.

File: modules/customize-certificates-api-add-named.adoc [hunk 2]
```diff
@@ -78,3 +78,21 @@ spec:
...
+$ oc get clusteroperators kube-apiserver
+----
++
+Do not continue to the next step until `PROGRESSING` is listed as `False`, as shown in the following output:
++
+.Example output
+[source,terminal]
+----
+NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
+kube-apiserver   4.7.0     True        False         False      145m
```
Comment (xingxingxia) [inline modules/customize-certificates-api-add-named.adoc:95]: s/4.7.0/{product-version}.0/ if you want quicker backports

Comment (bparry02) [reply modules/customize-certificates-api-add-named.adoc]: I have added the version variable.

File: modules/customize-certificates-api-add-named.adoc (hunk 1 above)
Comment (bparry02) [reply modules/customize-certificates-api-add-named.adoc:87]: I have added a statement that the operator may take a moment to start the rollout.

**Comment by bparry02** [comment]
Preview is showing the `product-version` variable being resolved to `Branch Build`. Hope that's expected.

**Comment by bparry02** [comment]
I have addressed the comments and this is ready for review.

**Pull request merged**